### ISSUE TITLE
Links page color contrast issue with dark mode

### ISSUE THREAD

**Issue by bennettforkner** [gordon-cs/gordon-360-ui#2270]
The text on some pages in dark mode does not follow requirements for web accessibility (and is difficult to read aside from that). Specifically, the links on the links page are not within regulation for appropriate color contrast ratios.

The colors in question are the dark mode values for `--mui-palette-link-main` and `--mui-palette-background-paper`. `--mui-palette-primary-main` is also a concern, but I haven't found places where it is actively used.

Color contrast details/checker: https://webaim.org/resources/contrastchecker/?fcolor=0260AD&bcolor=151515

![Image](https://github.com/gordon-cs/gordon-360-ui/assets/43764277/88ad32a2-c052-4df6-8761-3518b00009ae)

**Comment by Luke-W-Hart** [comment]
@bennettforkner , Isaiah Diaz (@Dopey12) and I are going to possibly have a meeting with Stephen Dagley and his team to discuss possible ways to resolve this, as changing the color of these links affects links on all pages (Links, MyProfile, Involvements, Help, About).

Let us know if you have any thoughts or questions!

**Comment by Luke-W-Hart** [comment]
The idea for now is to change the main link color to a ScottieCyan which looks like this...

![Image](https://github.com/gordon-cs/gordon-360-ui/assets/123050501/a9a89598-1945-45c5-b6e7-a1cb8d826187)



![Image](https://github.com/gordon-cs/gordon-360-ui/assets/123050501/1e09f96a-e306-42ce-be91-1708fbbd6c85)

ScottieCyan does not pass all the contrast tests with the Gray in dark mode though, but making the Cyan slightly lighter passes. The hover is also ScottieCyan so we are just trying out a few different things to resolve this.

**Comment by Dopey12** [comment]
On top of that I also not that secondary color was Gordon Blue, contrast wise it didn't do too well and overall it made certain things especially on Rec-Im is difficult to see. I believe the ScottieCyan with contrast text black looks better, it is the same as white mode but I think it fits dark mode good as well.


![Image](https://github.com/gordon-cs/gordon-360-ui/assets/156944565/203c1ef0-a753-4997-9ae3-3cf550af13b5)



![Image](https://github.com/gordon-cs/gordon-360-ui/assets/156944565/8edb9031-4391-4289-acdf-c6804965b1f7)



![Image](https://github.com/gordon-cs/gordon-360-ui/assets/156944565/5df0f0d3-3321-459c-b285-4f254e89ff70)
